openedx2zim/scraper.py

```diff
@@ -831,6 +842,24 @@ def render(self):
             self.build_dir.joinpath("assets"),
         )
 
+            
+    def handle_descriptions(self, default_description, description=None, long_description=None):
```

Comment:
Instead of this custom code, please use the one from https://github.com/openzim/python-scraperlib/blob/4dc30126a54040b4383ffed3617a1394a51b5a78/src/zimscraperlib/inputs.py#L56

Reply:
I have handled this change